client/app/rules/component/RuleList.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { MdPushPin } from 'react-icons/md'
+import { useGetHouseQuery } from '@/_hooks/api'
+
+const RuleList = () => {
+  const { data: house } = useGetHouseQuery()
+
+  return (
+    <>
+      <div className='mt-6 border border-amber-400 bg-amber-100 md:mt-8'>
+        {house?.rules.map((rule, index) => (
+          <div
+            key={index}
+            className='mx-4 my-7 flex items-center md:mx-6 md:my-8 '
+          >
```

Comment:
Because there is id for rule, avoid using index as key!
And, the container has py-4 px-2 for sp and py-7 px-7 for desktop. It is better to have padding in container and each one controlled by flex gap. So you don't need to margin every single rule!